src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -229,6 +231,28 @@ public static String getListDefaultDatabaseValue() {
 
     private final boolean standalone;
 
+    private final Function<ProductContentInfo, ProductContent> productContentInfoToProductContent =
```

Comment:
if we're going declare this as an instance variable, we can just declare it as a function instead:

```
private ProductContent translateProductContent(ProductContentInfo pcinfo) {
  ...
}
```